content/07.supplemental.md

```diff
@@ -11,8 +11,10 @@ Figure created with biorender.com.
 **Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.**  
 Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. 
 Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. 
-Upset plot (F) showing overlap of variant calls. 
-Cumulative distribution TMB plots for PBTA (G) and TCGA (H)tumors using consensus SNV calls.
+Upset plot (F) showing overlap of variant calls.
+Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same samples (N = ).
+<!-- TODO: come back to this N. -->
```

Comment:
I just opened this PR to track `N` more clearly: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1296

Reply:
Thanks - 98 samples from 13 patients seems like a high number of samples to me. I may hold on this until that PR is in.

I did some fuzzy math, and checked some of these numbers - we might want to be sure we have matches on at least `tumor_descriptor`. Anything below with N = 1 has a sample only in WGS only, and this accounts for 11 samples out of 52 in the `bs` dataframe below, which is much less than the 98 above.

```r
> matched_participants <- v21 %>%
+   filter(experimental_strategy != "RNA-Seq") %>%
+   group_by(Kids_First_Participant_ID) %>%
+   summarize(strategies = paste0(experimental_strategy, collapse = ",")) %>%
+   filter(grepl("WXS", strategies) & grepl("WGS", strategies)) %>%
+   pull(Kids_First_Participant_ID) 

> # Get the biospecimen IDS for these participants.
> bs <- v21 %>%
+   filter(Kids_First_Participant_ID %in% matched_participants) %>%
+   filter(experimental_strategy != "RNA-Seq" & !is.na(pathology_diagnosis)) %>%
+   select(Kids_First_Participant_ID, Kids_First_Biospecimen_ID, tumor_descriptor, experimental_strategy) %>%
+   unique() %>%
+   mutate(pt_desc = paste(Kids_First_Participant_ID, tumor_descriptor, sep = "_")) %>%
+   group_by(pt_desc, experimental_strategy) %>%
+   tally()

> as.data.frame(table(bs$pt_desc))
                                          Var1 Freq
1                PT_0MXPTTM3_Initial CNS Tumor    2
2                PT_1E3E6GMF_Initial CNS Tumor    2
3                PT_9GKVQ9QS_Initial CNS Tumor    2
4                PT_HGM20MW7_Initial CNS Tumor    2
5                PT_KBFM551M_Initial CNS Tumor    2
6  PT_KBFM551M_Progressive Disease Post-Mortem    1
7                PT_KTRJ8TFY_Initial CNS Tumor    2
8  PT_KTRJ8TFY_Progressive Disease Post-Mortem    1
9                PT_KZ56XHJT_Initial CNS Tumor    2
10                     PT_KZ56XHJT_Progressive    2
11 PT_KZ56XHJT_Progressive Disease Post-Mortem    1
12               PT_M23Q0DC3_Initial CNS Tumor    2
13               PT_M9XXJ4GR_Initial CNS Tumor    2
14               PT_NK8A49X5_Initial CNS Tumor    2
15                     PT_NK8A49X5_Progressive    2
16               PT_QA9WJ679_Initial CNS Tumor    2
17               PT_VPEMAQBN_Initial CNS Tumor    2
18               PT_WGVEF96B_Initial CNS Tumor    2
```

Reply:
I calculated those numbers based on the code that is used to create the figure itself, so we can chat about it over in that PR, and whether the figures themselves are accurate.

Reply:
sounds good!